From 858e56f551db26c96551e6e2b969a817e97ceb70 Mon Sep 17 00:00:00 2001
From: Olivier Moysan <olivier.moysan@foss.st.com>
Date: Wed, 12 Apr 2023 10:36:33 +0200
Subject: [PATCH 0391/1141] arm64: dts: st: change of sai4a dma config on
 stm32mp251

Change SAI4 DMA configuration to avoid DMA resource conflicts
on STM32MP257F-VALID3 board using SNOR_SNAND configuration.
There are not enough DMA channels with 32 bytes fifo.
Use DMA with 128 bytes fifo instead.
This change is required while smart DMA channel allocation is not
implemented. It can be dropped when DMA channel dispatcher is available.

Signed-off-by: Olivier Moysan <olivier.moysan@foss.st.com>
Change-Id: I9d71ce0b304a54034aec2ce5b45e57c820b51f92
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/312144
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
Domain-Review: Arnaud POULIQUEN <arnaud.pouliquen@st.com>
---
 arch/arm64/boot/dts/st/stm32mp251.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/arm64/boot/dts/st/stm32mp251.dtsi
+++ b/arch/arm64/boot/dts/st/stm32mp251.dtsi
@@ -1281,7 +1281,7 @@
 					#sound-dai-cells = <0>;
 					clocks = <&rcc CK_KER_SAI4>;
 					clock-names = "sai_ck";
-					dmas = <&hpdma 79 0x43 0x21 0>;
+					dmas = <&hpdma 79 0x63 0x21 0>;
 					status = "disabled";
 				};
 
